src/widgets/AppWindow.js

```diff
@@ -2099,194 +2098,51 @@ export default class AppWindow extends createjs.Container {
 
   _showGridTestCase() {
     this._clearScreen();
```

Comment:
Refactored this test case to be closer to a real grid use case by making it like tic-tac-toe, but time wasn't available to fully implement that game, so turn swapping and victory/draw conditions aren't implemented.